src/mcts/node.h

```diff
@@ -86,6 +86,33 @@ class Edge {
   float GetP() const;
   void SetP(float val);
 
+  void MakeTerminal(GameResult result);
+  void MakeCertain(GameResult result, CertaintyTrigger trigger);
+  void MakeCertain(float q, CertaintyTrigger trigger);
+  void SetEQ(float eq);
+  void ClearEdge() { certainty_state_ = 0; };
```

Comment:
Poor name. It only clears certainty, not entire edge.

Reply:
Any ideas for a name. Maybe ClearEdgeCertaintyStatus ?